### ISSUE TITLE
nan loss

### ISSUE THREAD

**Issue by kurbobo** [NVIDIA/flowtron#42]
I'm trying to train flowtron on LibriSpeech, I'm making everything as it is in repository, but there is always such result:
`10232:  nan
WARNING:root:NaN or Inf found in input tensor.
/opt/conda/conda-bld/pytorch_1579040055865/work/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/opt/conda/conda-bld/pytorch_1579040055865/work/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.`
is it okey? if it's not, what am I doing wrong? (if the path to data samples was wrong, there would have been an error, and there is no error, and all input values doesn't contain nan or inf)

**Comment by rafaelvalle** [comment]
Change `byte` to `bool` https://github.com/NVIDIA/flowtron/blob/6e43fc2e5792fe95c7c58e764d19843e3be308f8/flowtron.py#L33

**Comment by kurbobo** [comment]
thanks a lot!
